python/hail/docs/tutorials/09-matrixtable.ipynb

```diff
@@ -0,0 +1,476 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# MatrixTable\n",
+    "\n",
+    "If you've gotten this far, you're probably thinking:\n",
+    " - \"Can't I do all of this in `pandas` or `R`?\" \n",
+    " - \"What does this have to do with biology?\"\n",
+    "\n",
+    "Fear not!  You've learned most of the basic concepts of Hail and now you're ready for the bit that makes it easy to work with genetic data: the [MatrixTable](https://hail.is/docs/devel/hail.MatrixTable.html)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "In the last example, the ratings table had two keys: `movie_id` and `user_id`.  The ratings were secretly a movie by user matrix!\n",
+    "\n",
+    "Since rating is very sparse, it might be reasonable to store in this way in so-called \"coordinate form\" where each rating is a row of a table.\n",
```

Comment:
Since ratings are very sparse, it's reasonable to store them in so-called “coordinate form” where each rating is a row of a table.

Reply:
You missed this.